holidays/countries/bulgaria.py

```diff
@@ -43,13 +44,25 @@ class Bulgaria(HolidayBase, ChristianHolidays, InternationalHolidays):
 
     country = "BG"
     default_language = "bg"
+    # %s (Observed).
+    observed_label = tr("%s (почивен ден)")
     supported_categories = {PUBLIC, SCHOOL}
     supported_languages = ("bg", "en_US", "uk")
 
     def __init__(self, *args, **kwargs):
         ChristianHolidays.__init__(self, JULIAN_REVISED_CALENDAR)
         InternationalHolidays.__init__(self)
-        super().__init__(*args, **kwargs)
+        super().__init__(
+            observed_rule=SAT_SUN_TO_NEXT_WORKDAY, observed_since=2017, *args, **kwargs
+        )
+
+    def _populate_observed(self, dts: Set[date], multiple: bool = False) -> None:
```

Comment:
`multiple` is unused

Reply:
I just left parent class method signature unchanged 👀